frontend/src/scenes/insights/InsightsTable/InsightsTable.tsx

```diff
@@ -139,20 +141,25 @@ export function InsightsTable({ isLegend = true, showTotalCount = false }: Insig
     })
 
     if (indexedResults?.length > 0 && indexedResults[0].data) {
-        const valueColumns: ColumnsType<IndexedTrendResult> = indexedResults[0].data.map(({}, index: number) => ({
-            title: (
-                <DateDisplay
-                    interval={(filters.interval as IntervalType) || 'day'}
-                    date={(indexedResults[0].dates || indexedResults[0].days)[index]}
-                    hideWeekRange
-                />
-            ),
-            render: function RenderPeriod({}, item: IndexedTrendResult) {
-                return maybeAddCommasToInteger(item.data[index])
-            },
-            sorter: (a, b) => a.data[index] - b.data[index],
-            align: 'center',
-        }))
+        const valueColumns: ResizableColumnType<IndexedTrendResult>[] = indexedResults[0].data.map(
+            ({}, index: number) => ({
+                title: (
+                    <DateDisplay
+                        interval={(filters.interval as IntervalType) || 'day'}
+                        date={(indexedResults[0].dates || indexedResults[0].days)[index]}
+                        hideWeekRange
+                    />
+                ),
+                render: function RenderPeriod({}, item: IndexedTrendResult) {
+                    return maybeAddCommasToInteger(item.data[index])
+                },
+                sorter: (a, b) => a.data[index] - b.data[index],
+                align: 'center',
+                span: 1,
+                widthConstraints: [96, Infinity],
+                defaultWidth: 150,
```

Comment:
`150px` for some reason translates into `96px` on initial rendering. Any other value causes the columns to jump.